### PR TITLE
Release: v1.2

### PR DESCRIPTION
**Pull request by sgammon**
## Description
This changeset prepares release `v1.2`, in which schema is updated to `master` again after the `1.5.0` release, and other various small cleanup items have been completed.

Worklog:
- [x] Upgrade schema to `v1.5.0` on master
- [x] Upgrade POM dependencies and plugins
- [x] Cut and publish release

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [submodule "schema"]
 	path = schema
 	url = git@github.com:bloombox/schema.git
-	branch = release/1.5.0
+	branch = master
```
Comment (sgammon) [inline .gitmodules:4]: upgraded schema, now that `1.5.0` is out

File: Makefile [hunk 2]
```diff
@@ -19,8 +19,8 @@
 
 BUILDMODE ?= maven
 TESTS ?= yes
-RELEASE_VERSION ?= 1.1
-CLIENT_VERSION ?= 1.1-SNAPSHOT
+RELEASE_VERSION ?= 1.2
+CLIENT_VERSION ?= 1.2-SNAPSHOT
```
Comment (sgammon) [inline Makefile:23]: obligatory version bump

File: pom-embedded.xml [hunk 3]
```diff
@@ -82,10 +82,10 @@
         <bloombox.telemetry.endpoint>telemetry.api.bloombox.cloud</bloombox.telemetry.endpoint>
 
         <!-- Dependencies -->
-        <grpc.version>1.8.0</grpc.version>
+        <grpc.version>1.11.0</grpc.version>
         <protobuf.version>3.5.1</protobuf.version>
-        <netty.boringssl.version>2.0.6.Final</netty.boringssl.version>
-        <kotlin.version>1.2.21</kotlin.version>
+        <netty.boringssl.version>2.0.7.Final</netty.boringssl.version>
+        <kotlin.version>1.2.31</kotlin.version>
```
Comment (sgammon) [inline pom-embedded.xml:88]: updated dependencies

**Review by tyler-porras — APPROVED** [review]

**Review by RandyStevens — APPROVED** [review]
mergebuttonyes

**Review by mstan — APPROVED** [review]

**Pull request merged**